pack/eoe/eoep.json

```diff
@@ -823,76 +1090,306 @@
-        "traits": "Item. Tome.",
+        "subname": "Symbol of Power",
+        "text": "Exceptional.\nWhile you have...\n- ...10+ resources, you get +1 health and +1 sanity.\n- ...15+ resources, you may take one additional action during your turn.\n- ...20+ resources, you get +1 to each of your skills.",
+        "traits": "Item. Relic.",
         "type_code": "asset",
         "xp": 3
     },
     {
-        "code": "08071",
+        "code": "08058",
+        "cost": 2,
+        "deck_limit": 1,
+        "exceptional": true,
+        "faction_code": "rogue",
+        "illustrator": "Robert Laskey",
+        "is_unique": true,
+        "name": "The Red Clock",
+        "pack_code": "eoep",
+        "position": 58,
+        "quantity": 1,
+        "skill_wild": 2,
+        "slot": "Accessory",
+        "subname": "Broken but Reliable",
+        "text": "Exceptional. Uses (0 charges).\n<b>Forced</b> - After your turn begins: You may take all charges here, as resources. Place 1 charge here. If it has exactly...\n- 1 charge, you get +4 skill value for your next skill test.\n- 2 charges, you may move up to 3 times.\n- 3 charges, you may take 2 additional actions this turn.",
+        "traits": "Item. Relic.",
+        "type_code": "asset",
+        "xp": 5
+    },
+    {
+        "code": "08059",
+        "cost": null,
+        "deck_limit": 1,
+        "faction_code": "mystic",
+        "illustrator": "Derek D. Edgell",
+        "name": "Down the Rabbit Hole",
+        "pack_code": "eoep",
+        "permanent": true,
+        "position": 59,
+        "quantity": 1,
+        "text": "Permanent. Limit 1 per deck. Purchase at deck creation.\nAfter each scenario of a campaign, reduce the experience cost of the first 2 cards you upgrade before the next scenario by 1.\nIncrease the experience cost for you to purchase new cards by 1.",
+        "traits": "Talent.",
+        "type_code": "asset",
+        "xp": 0
+    },
+    {
+        "code": "08060",
+        "cost": 3,
+        "deck_limit": 2,
+        "faction_code": "mystic",
+        "illustrator": "Duallbrush",
+        "name": "Dragon Pole",
+        "pack_code": "eoep",
+        "position": 60,
+        "quantity": 2,
+        "skill_combat": 1,
+        "slot": "Hand x2",
+        "text": "You have 1 additional arcane slot.\n[action]: <b>Fight.</b> You get +1 [combat] for this attack for each of your arcane slots that is filled. If at least 2 of your arcane slots are filled, this attack deals +1 damage.",
+        "traits": "Item. Weapon. Melee.",
+        "type_code": "asset",
+        "xp": 0
+    },
+    {
+        "code": "08061",
+        "cost": 2,
+        "deck_limit": 2,
+        "faction_code": "mystic",
+        "illustrator": "Mauro Dal Bo",
+        "flavor": "You let your mind drift into cold, meaningless oblivion.\nThe emptiness feels like home.",
```

Comment:
No need for a linebreak here - it's just the formatting on the card rather than a forced break.